workitem/workitem_repository.go

```diff
@@ -538,7 +538,7 @@ func (r *GormWorkItemRepository) listItemsFromDB(ctx context.Context, criteria c
 		}
 		db = db.Limit(*limit)
 	}
-	db = db.Select("count(*) over () as cnt2 , *")
+	db = db.Select("count(*) over () as cnt2 , *").Order("execution_order")
```

Comment:
Right. And based on the ordering you are doing in the SQL above, ascending, the result from GET /api/workitems is:

```
postgres=# SELECT count(*) over () as cnt2 , fields->>'system.title', execution_order FROM "work_items"  WHERE "work_items".deleted_at IS NULL AND (('true')) ORDER BY execution_order LIMIT 22;                  
 cnt2 | ?column? | execution_order 
------+----------+-----------------
    3 | Test 1   |            1000
    3 | Test 2   |            2000
    3 | Test 3   |            3000
(3 rows)
```

To get the correct ordering you need to do: 

```go
Order("execution_order desc")
```

Agree?